### PR TITLE
tasks are sorted according to title if the days to complete is the same

### PR DESCRIPTION
**Pull request by yaseenagag**
Fixes # .

## Overview

Tasks are now sorted by days to complete first and if they clash they sort by title. 
files changed are template-task.js under queries

## Data Model / DB Schema Changes
N/A

## Environment / Configuration Changes
N/A

## Notes
N/A

### CODE REVIEW

File: src/utils/admins.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const ADMIN_GITHUB_HANDLES = ['shereefb', 'carlabagdonas', 'needdra', 'punitrathore']
+const ADMIN_GITHUB_HANDLES = ['shereefb', 'carlabagdonas', 'needdra', 'punitrathore', 'yaseenagag']
```
Comment (punitrathore) [inline src/utils/admins.js:1]: This shouldn't be checked in.

File: src/database/queries/template_task.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ const deleteAll = () =>
 const getAll = () =>
   _.findAll( 'template_task' )
   .orderBy('days_to_complete', 'asc')
+  .orderBy('title', 'asc')
```
Comment (punitrathore) [inline src/database/queries/template_task.js:16]: Can you write a test that covers this edge case?

File: src/utils/admins.js (hunk 1 above)
Comment (yaseenagag) [reply src/utils/admins.js:1]: what do you mean, i am not supposed to add my name to the master branch??
